index.html

```diff
@@ -777,7 +777,28 @@ <h2>Privacy Considerations</h2>
             This section details the general privacy considerations and specific privacy
             implications of deploying this specification into production environments.
          </p>
-         <section class="informative">
+         <section>
+            <h3>Personally Identifiable Information</h3>
+            <p>
+               Data associated with schemas and <a>verifiable credentials</a> are susceptible
+               to privacy violations when shared. Personally identifying data, such as a
+               government-issued identifier, address, or name, can be used to track and correlate
+               entities. Even less overt personal data such as a birthdate or postal code has
+               the ability to result in correlation and de-anonymization.
+            </p>
+            <p>
+               Implementers are strongly advised to avoid constructing schemas with any personally
+               identifiable information.
```

Comment:
I know the `PII` abbreviation is frowned on in some arenas, but it's in common use, and is likely to be a search term -- which would not now hit this document. Hence...
```suggestion
               identifiable information (PII).
```